core/renderers/zelos/drawer.js

```diff
@@ -59,20 +60,20 @@ Blockly.zelos.Drawer.prototype.drawTop_ = function() {
   this.outlinePath_ +=
       Blockly.utils.svgPaths.moveBy(topRow.xPos, this.info_.startY);
   for (var i = 0, elem; (elem = elements[i]); i++) {
-    if (elem.type == 'round corner') {
+    if (elem.type & Blockly.blockRendering.Types.LEFT_ROUND_CORNER) {
```

Comment:
Why do you sometimes use `&` directly and sometimes use it through the helper methods on `Blockly.blockRendering.Types`?

Reply:
I shouldn't have.. Will check and update all instances of this.

Reply:
Fixed.